pandas/plotting/_core.py

```diff
@@ -334,7 +334,11 @@ def result(self):
     def _compute_plot_data(self):
         data = self.data
 
-        if isinstance(data, Series):
+        from pandas import Series
```

Comment:
New hypothesis: `Series._set_subtyp` can sometimes set `_subtyp` to `time_series`, in which case it will not get recognized as `ABCSeries`.  It tentatively looks like the Travis errors are date-related cases.